pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -106,7 +106,8 @@
         "1.0.1",
         "1.0.2",
         "1.1.0",
-        "2.0.0"
+        "2.0.0",
+        "2.0.1"
```

Comment:
Is this really the only stable package? We didn't have anything else outside of the NETCoreApp package?

Reply:
From as far as I know we don't I can go over through the commits and validate that nobody forgot to add a package to the list, but currently in the list this is the only package.

Reply:
There where changes in Microsoft.CSharp and System.Security.Cryptography.OpenSsl but they are both marked as <IsNETCoreApp>true</IsNETCoreApp> and they are part of the metapackage.